src/db/StockPathHelper.js

```diff
@@ -0,0 +1,71 @@
+import { adultImageNames, kidImageNames } from './imageNames';
+
+function adultImagesPath() {
+  const domain = window.location.href;
+  if (domain.includes('localhost:3000') || domain.includes('sanapolku')) {
+    return 'sanapolku/data/paths/adult/images';
+  }
+  return 'public/data/paths/adult/images'; // Path for testing
+}
+
+function kidImagesPath() {
+  const domain = window.location.href;
+  if (domain.includes('localhost:3000') || domain.includes('sanapolku')) {
+    return 'sanapolku/data/paths/kid/images';
+  }
+  return 'public/data/paths/kid/images'; // Path for testing
+}
+
+export function getAdultPath() {
+  let words = shuffle(adultImageNames);
+  if (arguments.length === 1) {
+    words = words.slice(0, arguments[0]);
+  }
+  const path = adultImagesPath();
+  return words.map((fileName) => {
+    const imagePath = `${path}/${fileName}`;
+    const parsed = parseName(fileName);
+    return {
+      ...parsed,
+      img: imagePath,
+    };
+  });
+}
+
+export function getKidPath() {
+  let words = shuffle(kidImageNames);
+  if (arguments.length === 1) {
+    words = words.slice(0, arguments[0]);
+  }
+  const path = kidImagesPath();
+  return words.map((fileName) => {
+    const imagePath = `${path}/${fileName}`;
+    const parsed = parseName(fileName);
+    return {
+      ...parsed,
+      img: imagePath,
+    };
+  });
+}
+
+function shuffle(array) {
+  const shuffled = array.slice();
+  for (let i = shuffled.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [shuffled[i], shuffled[j]] = [shuffled[j], shuffled[i]];
+  }
+  return shuffled;
+}
```

Comment:
Meiltä löytyy jo kirjasto `import shuffleArray from 'lodash.shuffle';`
Käytetään sitä mieluummin kuin omaa toteutusta random shufflesta. Vaihda tämän funktion toteutus käyttämään tuota.

Reply:
Tein juuri näin